components/sidebar/sidebar_channel/sidebar_base_channel/sidebar_base_channel.tsx

```diff
@@ -83,6 +84,7 @@ export default class SidebarBaseChannel extends React.PureComponent<Props, State
                 closeHandler={this.getCloseHandler()!}
                 icon={this.getIcon()!}
                 isCollapsed={this.props.isCollapsed}
+                hasDraft={hasDraft}
```

Comment:
Passing `hasDraft` through every component type corresponding to a channel type adds a bit of code bloat, and sort of violates DRY.

Since none of those components actually need `hasDraft`, I propose we just use the redux connector for `SidebarChannelLink` to pass in `hasDraft`.

Reply:
I think I resolved this, I removed it from the components that weren't using it and added it to the connector in `SidebarChannelLink`